storage/tables/data-types/index.md

```diff
@@ -13,7 +13,7 @@ example, the transformation [`COPY` mapping](/transformations/snowflake/#load-ty
 the transformations. Also, some writers, e.g., the [Snowflake writer](/components/writers/database/snowflake/) use
 the table metadata to [pre-fill the table columns](/components/writers/database/snowflake/#table-configuration) configuration for you.
 
-Even if a data type is available for a column, that column is always stored as text --- keep this in mind
+Even if a data type is available for a column, that column is always stored as text (unless [native datatypes](#native-datatypes) feature is enabled) --- keep this in mind

```

Comment:
Toto bych změnil celé. Native datatypes budů default v budoucnu.

- By default storage always creates internally all columns for table as text not null, null-able values are converted to empty strings (exception is Exasol backend which interprets empty strings as null, so everything is null)
- This behavior can be changed in certain cases with feature [native datatypes](#native-datatypes) which will be default in the future

Reply:
co bude v budoucnu je v budoucnu, tady by mělo být jak je to teď

Reply:
Teď platí, že sa to bude v budoucnu default :D

Reply:
- jak pise Odin, dokumentace by mela popisovat to co je ted. Default bych vypustil
- zkratil bych vysvetlovani situace kolem exasolu, tam tam jen `(except for Exasol where everything is null)`. Tim ze to je tak dlouhy, tak se zapomene puvodni myslenka